index.html

```diff
@@ -136,22 +136,6 @@ <h2>Trusted by industry leaders</h2>
       </div>
     </div>
   </div>
-
-  <div class="beta">
-    <div class="container">
-      <div class="row">
-        <div class="col-sm-12">
-          <p>
-            <b>Beta:</b>
-            When you build software with Bazel, you're running the same code that has been refined and tested for years at Google to build heavy-duty, mission-critical infrastructure, services, and applications.
```

Comment:
nit: I feel like we can re-purpose this well crafted line somewhere, instead of deleting it.

Reply:
I've put it back.
ptal

Reply:
lg